fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(res => res.json())
+      .then(res => this.setState({ users: res, isLoading: false }));
```

Comment:
Using `res` a second time works but can be a bit confusing, using a parameter name that fits the data received is a better way to go. Something like `users` in this case or even `data` would be good.